projects/storefrontapp-e2e-cypress/cypress/helpers/coupons/cart-coupon.ts

```diff
@@ -142,9 +142,11 @@ export function claimCoupon(couponCode: string) {
 const cartCouponInput = 'input.input-coupon-code';
 const cartCouponButton = 'button.apply-coupon-button';
 const applyCartCoupon = (code: string) => {
+  interceptVoucherPostRequest();
   cy.get('cx-cart-coupon').within(() => {
     cy.get(cartCouponInput).type(code);
     cy.get(cartCouponButton).click();
```

Comment:
Click on 'Apply' Coupon always generated POST occ vouchers call, adding a wait command to avoid race condition.